eland/operations.py

```diff
@@ -1408,3 +1438,18 @@ def es_info(self, query_compiler, buf):
     def update_query(self, boolean_filter):
         task = BooleanFilterTask(boolean_filter)
         self._tasks.append(task)
+
+    @staticmethod
+    def quantile_to_percentile(quantile: Any) -> float:
```

Comment:
I tend to dislike `staticmethods`, these are almost always better off as just a function.

Also since we're raising a `TypeError` for any non-`float`/`int` shouldn't `quantile` be `Union[float, int]`?

Reply:
Yep that's correct on typing, and agree on `staticmethods`